src/pathfinding_igvc/src/PathFinderNode.cpp

```diff
@@ -0,0 +1,79 @@
+/*
+ * Created By: Min Gyo Kim
+ * Created On: May 23rd 2018
+ * Description: Implementation of Path Finder Node
+ */
+
+#include <PathFinderNode.h>
+
+PathFinderNode::PathFinderNode(int argc, char** argv, std::string node_name) {
+    ros::init(argc, argv, node_name);
+    ros::NodeHandle nh;
+    ros::NodeHandle private_nh("~");
+
+    SB_getParam(private_nh,
+                std::string("global_frame_name"),
+                this->_global_frame_name,
+                std::string("/map"));
+    SB_getParam(private_nh,
+                std::string("base_frame_name"),
+                this->_base_frame_name,
+                std::string("/base_link"));
+
+    std::string grid_subscriber_topic = "/occupancy_grid"; // dummy topic name
```

Comment:
I get what you mean by `dummy topic name` here, but I think it could be a bit confusing, as most of our topics are "dummy" and get re-mapped, but we normally don't say that.